packages/core/src/components/forms/_controls.scss

```diff
@@ -131,9 +129,23 @@ $control-checked-background-color-active: nth(map-get($button-intents, "primary"
   */
 
   &.#{$ns}-checkbox {
+    @mixin indicator-icon($icon) {
+      &::before {
+        // embed SVG icon image as backgroud-image above gradient.
+        // the SVG image content is inlined into the CSS, so use this sparingly.
+        background-image: svg("16px/#{$icon}.svg", (path: (fill: $white)));
```

Comment:
why not just parameterize whole path? we'll never want 32px icons?

Reply:
we only have 16px and 20px icons and this way you can access either. in this case we don't need the 20px since the forms are so simple.